docs/blog/2023-07-28-release-v1.0.0/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Presenting Woodpecker 1.0.0
+description: Introducing Woodpecker 1.0.0 and its new features.
+slug: release-v1.0.0
+authors:
+  - name: 6543
+    title: Maintainer of Woodpecker
+    url: https://github.com/6543
+    image_url: https://github.com/6543.png
+tags: [release, stable]
+hide_table_of_contents: false
+---
+
+We are proud to present you Woodpecker v1.0.0.
+It took us quite some time, but now we are sure it's ready, and so you should really have a look at it.
+
+<!--truncate-->
+
+We've refactored a lot of code, so contributing to the codebase should be much easier.  
```

Comment:
```suggestion
We've refactored a lot of code and contributing to the codebase should be much easier now.
```

Reply:
well it could be better worded but I personally prefer the current wording over the suggested one

Reply:
Thats totally fine